packages/react-widgets/src/DropdownList.tsx

```diff
@@ -391,9 +401,17 @@ const DropdownListImpl: DropdownList = React.forwardRef(function DropdownList<
       if (altKey) return closeWithFocus()
 
       list.focus(list.prev())
+    } else if (key === 'Backspace' && allowClear && currentValue && !currentSearch) {
+      handleClear(e);
     }
   }
 
+  const handleClear = (e: React.MouseEvent<HTMLSpanElement> | React.KeyboardEvent<HTMLDivElement>) => {
+    e.preventDefault();
+    e.stopPropagation();
```

Comment:
is this to prevent toggling the dropdown open as well?

Reply:
exactly, yes